services/core-api/app/api/services/ams_api_service.py

```diff
@@ -0,0 +1,169 @@
+import requests
+import json
+
+from app.api.municipalities.models.municipality import Municipality
+from app.config import Config
+from app.date_time_helper import get_date_iso8601_string
+
+
+class AMSApiService():
```

Comment:
Would be awesome if we could add a test or two for this service. Even if it's just something like "given this example authorization - here's the payload in plain json of what gets posted to AMS"

Reply:
Nice suggestion. I will update.

Reply:
ooo yeah and I think I'd want a test ensuring that the authorization still gets saved in our db if AMS errors out for whatever reason (bad request, expired token)

Reply:
Updated with 2 unit tests for successful outcome and unsuccessful outcome.